assoc-aggregate/prepare_segments_1.py

```diff
@@ -1,36 +1,34 @@
+# Author: Ash O'Farrell (aofarrel@ucsc.edu)
+#
 # Notes:
 # 1. This needs to be run in Python2
-# 2. This expects the input files to be in the workdir
+# 2. This expects the input files to be in the workdir, or else it'll fail when making the zips
 
 #~{segments_file}
-IIsegments_fileII = "_test-data-and-truths_/assoc/segments.txt"
+IIsegments_fileII = "segments.txt"
 
 # ['~{sep="','" input_gds_files}']
-IIinput_gds_filesII = ["_test-data-and-truths_/assoc/1KG_phase3_subset_chr1.gds", "_test-data-and-truths_/gds/a_vcf2gds/1KG_phase3_subset_chr2_butwithadifferentname.gds", "_test-data-and-truths_/gds/a_vcf2gds/1KG_phase3_subset_chrX.gds"]
+IIinput_gds_filesII = ["1KG_phase3_subset_chr1.gds", "1KG_phase3_subset_chr2_butwithadifferentname.gds", "1KG_phase3_subset_chrX.gds"]
```

Comment:
If this is being run as a script, I wonder if these inputs should be passed as parameters.

Reply:
It is not being run as a script. It is a standalone version of what's in the WDL. These strings are used just to mimic actual files.